instrumentation/opentelemetry-instrumentation-aiohttp-server/src/opentelemetry/instrumentation/aiohttp_server/__init__.py

```diff
@@ -0,0 +1,255 @@
+import urllib
+from aiohttp import web
+from multidict import CIMultiDictProxy
+from timeit import default_timer
+from typing import Tuple, Dict, List, Union
+
+from opentelemetry import context, trace, metrics
+from opentelemetry.instrumentation.aiohttp_server.package import _instruments
+from opentelemetry.instrumentation.aiohttp_server.version import __version__
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.utils import http_status_to_status_code
+from opentelemetry.propagators.textmap import Getter
+from opentelemetry.propagate import extract
+from opentelemetry.semconv.trace import SpanAttributes
+from opentelemetry.semconv.metrics import MetricInstruments
+from opentelemetry.trace.status import Status, StatusCode
+from opentelemetry.util.http import get_excluded_urls
+from opentelemetry.util.http import remove_url_credentials
+
+
+_SUPPRESS_HTTP_INSTRUMENTATION_KEY = "suppress_http_instrumentation"
```

Comment:
Other instrumentations like botocore, requests, urllib3 and urllib import this:
`from opentelemetry.context import _SUPPRESS_HTTP_INSTRUMENTATION_KEY`

Reply:
Thanks for pointing it out.